src/components/marker-chart/Canvas.js

```diff
@@ -6,10 +6,10 @@
 import * as React from 'react';
 import withChartViewport from '../shared/chart/Viewport';
 import ChartCanvas from '../shared/chart/Canvas';
-import MarkerTooltipContents from '../shared/MarkerTooltipContents';
 import TextMeasurement from '../../utils/text-measurement';
 import { BLUE_40 } from '../../utils/colors';
 
+import type { Element as ReactElement } from 'react';
```

Comment:
nit: you can just use React.Element as we use import * as React from 'react' above.